lexicons/app/bsky/actor/defs.json

```diff
@@ -15,7 +15,11 @@
           "maxLength": 640
         },
         "avatar": { "type": "string" },
-        "viewer": {"type": "ref", "ref": "#viewerState"}
+        "viewer": {"type": "ref", "ref": "#viewerState"},
+        "labels": {
+          "type": "array",
+          "items": {"type": "ref", "ref": "com.atproto.label.label"}
```

Comment:
I think... this might actually be the only case where a record schema is used as a `ref` 🤔 

I'm not sure if we've exactly specified the semantics of that? ie if `$type` is required in that case.

I would think not. Similar to schema fragments, it's included in a union but not a ref...

Reply:
might want @pfrazee to take a look at this 🤔 

We include `$type` when we need to differentiate between two options, which we don't need to do here. But without `$type`, the record will hash to a different CID than expected

Reply:
(and i continue leaning towards making labels there own free-floating thing instead of a record in a repo 😉 )